src/middlewared/middlewared/plugins/account.py

```diff
@@ -1305,7 +1305,7 @@ async def do_home_copy(self, job, home_old, home_new, username, new_mode, uid):
         await perm_job.wait()
 
         command = f"/bin/cp -a {shlex.quote(home_old) + '/' + '.'} {shlex.quote(home_new + '/')}"
-        do_copy = await run(["/usr/bin/su", "-", username, "-c", command], check=False)
+        do_copy = await run(["/usr/bin/su", "-s", "/usr/sbin/bash", "-", username, "-c", command], check=False)
```

Comment:
Should be /usr/bin/bash

Reply:
Leave it to me to typo I one-line commit. :D